exercises/evaluation/ex_rnw/ex_train-test.Rnw

```diff
@@ -0,0 +1,27 @@
+In supervised learning, we typically assume that the data set $\D = \Dset$ originates from a data generating process $\Pxy$ in an i.i.d manner, i.e., $\D \sim \left(\Pxy\right)^n$. One could split data set $\D$ with $n$ observations into subsets $\Dtrain$ and $\Dtest$ of sizes $\ntrain$ and $\ntest$ with $\ntrain + \ntest = n$. Both subsets can be represented with index vectors $\Jtrain \in \JtrainSpace$ and $\Jtest \in \JtestSpace$, respectively. For such an index vector $J$, one can define a corresponding vector of labels $\yJ = \yJDef \in J^{(m)}$ and a corresponding matrix of prediction scores $\FJf = \FJfDef \in \R^{m\times g}$ for a model $f$. For regression tasks, $g = 1$ and $\FJf$ is a vector.
```

Comment:
Mach da mal mehr Zeilen draus in dem .Rnw file, dann kann ich das besser kommentieren :)

in jedem Fall: wieso ist \yJ \in J^{(m)}? Das ist einfach falsch, oder?

Reply:
Stimmt, tut mir leid dass das alles in einer Zeile war. Das war falsch, sorry. Muss \Yspace^m sein.